api/hardwaremanagement/v1alpha1/node.go

```diff
@@ -38,6 +38,11 @@ type BMC struct {
 // a node that will eventually be part of a deployment manager.
 type NodeStatus struct {
 	BMC *BMC `json:"bmc,omitempty"`
+
+	BootMACAddress string `json:"bootMACAddress,omitempty"`
+
+	HostName string `json:"hostName,omitempty"`
```

Comment:
`HostName` is typically spelled as `Hostname` with a lowercase `N` ... in fact the spelling in some of the other files uses `hostname` so we should try to be consistent.

Reply:
Changed.